src/s2/s2edge_crossings_test.cc

```diff
@@ -383,12 +384,14 @@ class GetIntersectionStats {
            "Method", "Successes", "Attempts", "Rate");
     for (int i = 0; i < kNumMethods; ++i) {
       if (tally_[i] == 0) continue;
-      printf("%10s %9d %5.1f%% %9d %5.1f%%  %5.1f%%\n",
-             S2::internal::GetIntersectionMethodName(
-                 static_cast<IntersectionMethod>(i)),
-             tally_[i], 100.0 * tally_[i] / total,
-             totals[i], 100.0 * totals[i] / total,
-             100.0 * tally_[i] / totals[i]);
+      // Use `StrFormat` to handle `string_view`.
+      puts(absl::StrFormat("%10s %9d %5.1f%% %9d %5.1f%%  %5.1f%%",
```

Comment:
Why puts? Why not `absl::PrintF()`

Reply:
Thanks. I didn't know it existed.  This will be fixed in a future update.